cloud/azure/database/mariadb/mode/connections.pm

```diff
@@ -0,0 +1,188 @@
+#
+# Copyright 2022 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package cloud::azure::database::mariadb::mode::connections;
+
+use base qw(cloud::azure::custom::mode);
+
+use strict;
+use warnings;
+
+sub get_metrics_mapping {
+    my ($self, %options) = @_;
+
+    my $metrics_mapping = {
+        'active_connections' => {
+            'output' => 'Active Connections',
+            'label'  => 'connections-active',
+            'nlabel' => 'azmariadb.connections.active.count',
+            'unit'   => '',
+            'min'    => '0'
+        },
+        'connections_failed' => {
+            'output' => 'Failed Connections',
+            'label'  => 'connections-failed',
+            'nlabel' => 'azmariadb.connections.failed.count',
+            'unit'   => '',
+            'min'    => '0'
+        },
+        'aborted_connections' => {
+            'output' => 'Aborted Connections',
+            'label'  => 'connections-aborted',
+            'nlabel' => 'azmariadb.connections.aborted.count',
+            'unit'   => '',
+            'min'    => '0'
+        },
+        'total_connections' => {
+            'output' => 'Total Connections',
+            'label'  => 'connections-total',
+            'nlabel' => 'azmariadb.connections.total.count',
+            'unit'   => '',
+            'min'    => '0'
+        }
```

Comment:
Are these metrics really supported by Microsoft ? I can't see it in their official documentation: https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftdbformariadbservers

Reply:
I have removed the metrics which are not available.